### DISCUSSION TITLE
Able to connect from android client but unable to accss internet from laptop

### DISCUSSION THREAD

**Discussion by digital-idiot** [wg-easy/wg-easy#379] [Q&A] [answered]
I am facing this weird problem, I am running `wg-easy` using following command:

```
docker run -d \
  --name=wg-easy \
  -e WG_HOST=<host-ip> \
  -e PASSWORD=<paasword> \
  -e WG_PERSISTENT_KEEPALIVE=10 \
  -e WG_DEFAULT_DNS=dns.adguard-dns.com \
  -e WG_MTU=1384
  -v ~/.wg-easy:/etc/wireguard \
  -p 51820:51820/udp \
  -p 51821:51821/tcp \
  --cap-add=NET_ADMIN \
  --cap-add=SYS_MODULE \
  --sysctl="net.ipv4.conf.all.src_valid_mark=1" \
  --sysctl="net.ipv4.ip_forward=1" \
  --restart unless-stopped \
  weejewel/wg-easy

```
Obviously the server is working fine as I can connect using my phone and access internet. However, when I connect to the server from my laptop, I am unable to access internet. Can't get `ping` from Google. I am clueless about the reason of this behavior. Some help will be appreciated.

**Comment by TextureGold** [comment, marked as answer]
Hi, you need to cast <host-ip> to the world with port 51820
Might help.
